lib/OpenAPI/Client.pm

```diff
@@ -94,6 +96,7 @@ sub _generate_method {
       Mojo::IOLoop->next_tick(sub { $self->$cb($tx) });
       return $self;
     }
+    warn "[".__PACKAGE__."] $path no error\n" if DEBUG;
```

Comment:
I thought https://github.com/mohawk2/openapi-client/blob/55ea04ecf42e7cd0e0b6cd809ff8566f2d9018e4/lib/OpenAPI/Client.pm#L156 did the same..?